File: core/src/main/java/vn/loitp/model/Loitp.java

package vn.loitp.model;

import java.util.List;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Loitp {

    @SerializedName("loitp")
    @Expose
    private List<App> loitp = null;

    public List<App> getLoitp() {
        return loitp;
    }

    public void setLoitp(List<App> loitp) {
        this.loitp = loitp;
    }

}

File: core/src/main/java/com/loitpcore/model/Loitp.kt
package com.loitpcore.model

import androidx.annotation.Keep
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import com.loitp.core.base.BaseModel

/**
 * Created by Loitp on 04,August,2022
 * Galaxy One company,
 * Vietnam
 * +840766040293
 * freuss47@gmail.com
 */
@Keep
class Loitp : BaseModel() {

    @SerializedName("loitp")
    @Expose
    var loitp: List<App>? = null
}
